### PR TITLE
feat(keyring-api)!: add `scopes` field to `KeyringAccount`

### PR DESCRIPTION
**Pull request by ccharly**
See:
- https://github.com/MetaMask/accounts-planning/issues/627

### CODE REVIEW

**Comment by ccharly** [comment]
@metamaskbot publish-previews

**Comment by ccharly** [comment]
@metamaskbot publish-previews

**Comment by ccharly** [comment]
@metamaskbot publish-previews

File: packages/keyring-api/src/btc/constants.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+/* istanbul ignore file */
+
+/**
+ * Scopes for Bitcoin account type. See {@link KeyringAccount.scopes}.
+ */
+export enum BtcScopes {
+  Namespace = 'bip122',
+  Mainnet = 'bip122:000000000019d6689c085ae165831e93',
+  Testnet = 'bip122:000000000933ea01ad0ee984209779ba',
```
Comment (darioAnongba) [inline packages/keyring-api/src/btc/constants.ts:9]:
```suggestion
  Testnet = 'bip122:000000000933ea01ad0ee984209779ba',
  Testnet4 = 'bip122:00000000da84f2bafbbc53dee25a72ae',
  Signet = 'bip122:00000008819873e925422c1ff0f99f7c',
  Regtest = 'bip122:regtest',
```

**Pull request merged**